Cargo.toml

```
@@ -17,11 +17,12 @@ name = "fjall"
 path = "src/lib.rs"
 
 [features]
-default = ["bloom", "single_writer_tx", "lz4"]
+default = ["bloom", "ssi_tx", "lz4"]
```

```suggestion
default = ["bloom", "single_writer_tx", "lz4"]
```
The default impl should stay single writer, because changing it would be a breaking change.

---

Fixed it!